scripts/cmscp.py

```diff
@@ -1134,6 +1135,7 @@ def update_exit_info(exit_info, exit_code, exit_msg = None, force = False):
                              'CRAB_StageoutPolicy', \
                              'CRAB_Destination', \
                              'CRAB_Dest', \
+                             'CRAB_Final_Dest', \
```

Comment:
I would probably take this opportunity to rename 'CRAB_Dest' to 'CRAB_TempDestDir' and instead of 'CRAB_Final_Dest' I wold use 'CRAB_DestDir'. What others think?